modules/ai/assets/js/editor/pages/form-layout/hooks/use-screenshot.js

```diff
@@ -9,11 +9,17 @@ const useScreenshot = ( type, onData ) => {
 
 	const layoutData = useLayoutPrompt( type, null );
 
-	const generate = ( prompt, prevGeneratedIds, signal ) => {
+	const generate = ( prompt, attachments, prevGeneratedIds, signal ) => {
 		setIsLoading( true );
 		setError( ERROR_INITIAL_VALUE );
 
-		return layoutData.send( prompt, prevGeneratedIds, signal )
+		const cleanAttachments = attachments.map( ( attachment ) => {
+			const cleanAttachment = { ...attachment };
+			delete cleanAttachment.previewHTML;
+			return cleanAttachment;
+		} );
```

Comment:
Maybe you can:

```suggestion
		const cleanAttachments = attachments.map( ( { previewHTML, ...rest ) => rest );
```

not sure what "clean" means though